app/main.py

```diff
@@ -1 +1,46 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        for skill in ["JavaScript", "HTML", "CSS"]:
+            self.learn_skill(skill)
```

Comment:
Use the extend() method or += operator to add multiple values to a single list

Reply:
It means that we will not use learn_skill method, I thought, that if we create this method,  we should use it,
But ok.